server/server.go

```diff
@@ -76,6 +76,9 @@ var (
 	publishPathRegex       = regexp.MustCompile(`^/[-_A-Za-z0-9]{1,64}/(publish|send|trigger)$`)
 
 	webConfigPath                                        = "/config.js"
+	webManifestPath                                      = "/manifest.webmanifest"
+	webServiceWorkerPath                                 = "/sw.js"
+	webRegisterServiceWorkerPath                         = "/registerSW.js"
```

Comment:
Could these just go in `/static/` somewhere? That'd make it easier, since we wouldn't need special routes for them. We could move in the build process.

Reply:
I believe the manifest has to be at that path. The service workers could possibly be moved, though I don't really want to configure it differently since there are a bunch of `scope` related gotchas that I don't fully understand yet - I think it could possibly limit the service worker to a sub path.

Reply:
I checked - I removed registerSW by inlining it, but both the manifest and sw.js have to be at the root.